content/control-api/testing-with-postman.textile

```diff
@@ -0,0 +1,84 @@
+---
+title: Testing with Postman
+meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically."
+meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
+section: control-api
+index: 30
+jump_to:
+  Help with:
+    - Importing into Postman#importing
+    - Sending a request#send-request
+    - See also#see-also
+---
+
+There are several ways you can try out the Control API. You can use a command line tool such as "Curl":https://curl.se/ or "HTTPie":https://httpie.io/ to test out various requests. Another way is to use a graphical tool such as Postman or Paw. This topic looks at testing with Postman.
```

Comment:
You say there are "several ways" here but only mention two. Perhaps you could say something like:

_The Control API is designed for programmatic access to your account. However, you can test individual requests using either a command line tool such as Curl, or a graphical tool such as Postman or Paw. This topic shows you how to make requests using Postman._

Reply:
Agreed, also added links for all tools.